HarrisMoviesApplication/ViewController.swift

```diff
@@ -8,13 +8,42 @@
 import UIKit
 
 class ViewController: UIViewController {
+    
+    struct CatFact: Decodable {
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view.
-        //creating
-    }
+        // MARK: - Properties
 
+        let text: String
+    }
```

Comment:
Lets move this struct to a separate file

Reply:
ok i will separate all my files efficiently from now